Utilities/bootstrap

```diff
@@ -860,6 +872,10 @@ def main():
                         help="Directory in which to put libSwiftPM modules")
     parser.add_argument("--libswiftpm-skip-c-headers", action="store_true",
                         help="Skip installing C headers in libSwiftPM install directory")
+    parser.add_argument("--arch",
+                        help="The architecture being built for")
+    parser.add_argument("--sdk",
+                        help="The sdk platform being built for. e.g. (macosx, linux or windows)")
```

Comment:
I don't think we should add these as flags. We are not really using them as arguments.